### ISSUE TITLE
Image not displaying due to mixed content error in Android webview.

### ISSUE THREAD

**Issue by prabakarviji** [facebook/react-native#8460]
When trying to load an image from("http") in secured web page source("https"), it is not displaying an image because of mixed content.As per as, mixed content source blocked by default in Android 5.0 + version.If it good to proceed, i will make PR?

**Comment by cuitianze** [comment]
same question +1

**Comment by lacker** [comment]
fixed

**Comment by xanview** [comment]
Was this fixed in react native - or somewhere else? - what is the fix, do I need to upgrade the version of React I'm using?

**Comment by lacker** [comment]
Try RN master and if I am wrong and it's not fixed there then please feel free to reopen or open a new issue!

**Comment by xanview** [comment]
I'm using WebView and still getting this :( -
```
Mixed Content: The page at 'https://testdomain/' was loaded over HTTPS, but requested an insecure image 'http://testdomain/recordings/584232ed3d1f9272f9dc415f/thumbnail'. This request has been blocked; the content must be served over HTTPS.
(index):1 Mixed Content: The page at 'https://testdomain/' was loaded over HTTPS, but requested an insecure image 'http://testdomain/recordings/584234033d1f9272f9dc4259/thumbnail'. This request has been blocked; the content must be served over HTTPS.
```

Does the fix not apply to WebView?

**Comment by mehraan** [comment]
The PR was never merged. See https://github.com/facebook/react-native/pull/8696.

**Comment by lacker** [comment]
My bad. Would you mind providing a sample project that reproduces this problem in a new issue?
